WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -48,5 +49,27 @@ private static void ModifyRegistry(bool runOnSystemStartup, string pathToExeFile
 				key.DeleteValue(nameof(WalletWasabi), false);
 			}
 		}
+
+		private static void ModifyMacOsLoginItems(bool runOnSystemStartup)
```

Comment:
Maybe we can unify the names of the functions like this:

```suggestion
        //  StartOn<OS>Startup  
		private static void StartOnMacStartup(bool runOnSystemStartup)
```

or in some similar fashion.